### DISCUSSION TITLE
AWS Config dependency?

### DISCUSSION THREAD

**Discussion by AlladiPO** [aws-solutions/workload-discovery-on-aws#542] [Q&A] [answered]
Not all customers have AWS Config enabled. So what kind of a critical dependency does this project have on AWS Config. It should be able to find the associations between resources in other ways, isn't it?

**Comment by svozza** [comment, marked as answer]
AWS Config is fundamental to how Workload Discovery works. At the time of writing, Workload Discovery supports 419 resource types, of which 408 come from Config. For the other 11 resource types we use the AWS SDK library to make `List*` and `Describe*` calls to discover them (you can see a list of those APIs in the [docs](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/supported-resources-1.html)).

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
